src/GraphBLAS-sharp.Backend/Constants/Constants.fs

```diff
@@ -0,0 +1,18 @@
+namespace GraphBLAS.FSharp
+
+[<RequireQualifiedAccess>]
+module Constants =
+    module PageRank =
+        /// <summary>
+        /// PageRank algorithms will finish then
+        /// difference of current and previous vectors
+        /// is less than accuracy
+        /// </summary>
+        let accuracy = 1e-6f
+        /// <summary>
+        /// Damping factor for PageRank algorithm
+        /// </summary>
+        let alpha = 0.85f
+
+    module Common =
```

Comment:
I don't know how convenient it is to move something that is used only in tests here. But okay

Reply:
Previously it was in Backend.Utils and used in SpMV for the reason that I forgot. I imagine we can use it somewhere if we know for sure that this is the optimal size for the algorithm

Reply:
It seems to me that the optimal size should be in benchmarks. And there will be several of them, depending on the hardware and other things. So that it is not just optimal, but everyone can check it relatively quickly in a particular case, just by cloning the repository and writing a couple of commands.  But that's sometime later.

If this was hardcoded in SpMV, then shouldn't we make a separate submodule for SpMV in the constants module? Just like you did for PageRank? 

It is better to use a constant from the SpMV module in tests than to use a constant from tests in SpMV.

His whole idea is just not to forget where the constants came from or what actions to take with them. In my opinion, putting it in a separate module with an obvious name while providing comments with the reason is very good.